include/sys/proc.h

```diff
@@ -49,6 +50,7 @@ struct proc {
   TAILQ_ENTRY(proc) p_child;  /* (a) link on parent's children list */
   thread_t *p_thread;         /* (@) the only thread running in this process */
   pid_t p_pid;                /* (!) Process ID */
+  struct cred *p_cred;        /* Process credentials */
   char *p_elfpath;            /* (!) path of loaded elf file */
```

Comment:
Please add an information about `p_cred`'s corresponding lock
For instance: if `p_cred` is protected by `p_lock`, change the comment to: `/* (@) Process credentials */`